src/naresh/Assignment_23.java

```diff
@@ -0,0 +1,39 @@
+package naresh;
+/*
+ * Write a program to find count of Uppercase, lowercase, digits and special
+ * characters.
+ * 
+ * String str = "TecHNo_cR6ediTs"; output : Uppercase -> 5 Lowercase -> 8 Digit
+ * -> 1 Special char count -> 1
+ */
+public class Assignment_23 {
+
+	void upper(String input) {
+		int ucount = 0;
+		int lcount = 0;
+		int dcount = 0;
+		int scount = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isUpperCase(input.charAt(index))) {
```

Comment:
It would be better if you create char ch = input.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch